presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -212,6 +212,9 @@
 
     //TODO: Prestissimo related session properties that are temporarily put here. They will be relocated in the future
     public static final String PRESTISSIMO_SIMPLIFIED_EXPRESSION_EVALUATION_ENABLED = "simplified_expression_evaluation_enabled";
+    public static final String SMART_SAMPLING_ENABLED = "smart_sampling_enabled";
+    public static final String SMART_SAMPLING_PERCENTAGE = "smart_sampling_percentage";
+    public static final String SMART_SAMPLING_FUNCTION = "smart_sampling_function";
```

Comment:
Since there's currently no other sampling function options, let's remove this parameter.

Reply:
There is - FB_LOGGER_SAMPLING.